app/assess/data.py

```diff
@@ -128,20 +132,38 @@ def get_round_with_applications(
     return None
 
 
+def get_score_and_justification(application_id, sub_criteria_id):
+    url = Config.ASSESSMENT_SCORES_ENDPOINT
+    params = {
+        "application_id": application_id,
+        "sub_criteria_id": sub_criteria_id,
+    }
+    response = requests.get(url, params)
+    # current_app.logger.info(
+    #     "Response from Assessment Store:" + response.content
+    # )
+    if response.status_code == 200:
+        return response.json()
+    else:
+        return None
```

Comment:
Can we raise an exception, or at least log at error level in the non 200 case? This will make the issue less opaque if the request fails. I think ideally `get_data` should do the same.

Reply:
have moved this logic over to get-data and we are now logging failed requests in there 👍🏾